astropy/modeling/tests/test_models.py

```diff
@@ -452,9 +451,31 @@ def test_deriv_1D(self, model_class, test_parameters):
         model_no_deriv = create_model(model_class, test_parameters,
                                       use_constraints=False)
 
+        # NOTE: PR 10644 replaced deprecated usage of RandomState but could not
+        #       find a new seed that did not cause test failure, resorted to hardcoding.
```

Comment:
Hm - seems like this should maybe become a TODO, with a new issue for @nden?

Reply:
Yeah, haven't gotten any replies here, so follow-up issue is the way to go.